ND Supporting Document 2_2.adoc

```diff
@@ -2660,80 +2660,118 @@ Remark: Some systems might require the presence of the SAN extension. In this ca
+. The evaluator shall verify that the TSS describes the key exchange parameters of the server Key Exchange message. The evaluator shall ensure these algorithms are consistent with the selected ciphersuites.
+
+*FCS_TLSS_EXT.1.5*
+
+[arabic, start=555]
+. The evaluator shall verify that the TSS describes if session resumption based on session IDs is supported (RFC 4346 and/or RFC 5246), if session resumption based on session tickets is supported (RFC 5077) and/or if session resumption according to RFC8446 is supported.
 . If session tickets are supported, the evaluator shall verify that the TSS describes that the session tickets are encrypted using symmetric algorithms consistent with FCS_COP.1/DataEncryption. The evaluator shall verify that the TSS identifies the key lengths and algorithms used to protect session tickets.
 . If session tickets are supported, the evaluator shall verify that the TSS describes that session tickets adhere to the structural format provided in section 4 of RFC 5077 and if not, a justification shall be given of the actual session ticket format.
 
+*FCS_TLSS_EXT.1.6*
+
+[arabic, start=556]
+. The evaluator shall verify that TSS describes whether the list of supported ciphersuites can be configured or not.
+
+*FCS_TLSS_EXT.1.7*
+
+[arabic, start=557]
+. The evaluator shall verify in the TSS that, for TLS 1.3, the TOE shall not permit out-of-band provisioning of pre-shared keys (PSKs) in the evaluated configuration.
+
 ===== Guidance Documentation
 
 *FCS_TLSS_EXT.1.1*
 
 [arabic, start=557]
-. The evaluator shall check the guidance documentation to ensure that it contains instructions on configuring the TOE so that TLS conforms to the description in the TSS (for instance, the set of ciphersuites advertised by the TOE may have to be restricted to meet the requirements).
+. The evaluator shall check the guidance documentation to ensure that it contains instructions on configuring the TOE so that TLS conforms to the description in the TSS (for instance, the set of ciphersuites advertised by the TOE or TLS version supported by the TOE may have to be restricted to meet the requirements).
 
-*FCS_TLSS_EXT.1.2*
+*FCS_TLSS_EXT.1.3*
 
 [arabic, start=558]
 . The evaluator shall verify that any configuration necessary to meet the requirement must be contained in the AGD guidance.
 
-*FCS_TLSS_EXT.1.3*
+*FCS_TLSS_EXT.1.4*
 
-[arabic, start=559]
+[arabic, start=558]
 . The evaluator shall verify that any configuration necessary to meet the requirement must be contained in the AGD guidance.
 
+*FCS_TLSS_EXT.1.6*
+
+[arabic, start=559]
+. If the TSF provides the ability of configuring the list of supported ciphersuites, the evaluator shall verify that AGD guidance includes configuration of the list of supported ciphersuites.
+
+*FCS_TLSS_EXT.1.7*
```

Comment:
This should be FCS_TLSS_EXT.1.8.

Reply:
@plughy2 There is not a FCS_TLSS_EXT.1.8 in the most recent proposal document. FCS_TLSS_EXT.1.7 is correct here.